src/tests/run.sh

```diff
@@ -39,47 +39,12 @@ function print_usage {
     echo '  --limitedDumpGeneration          : '
 }
 
-function check_cpu_architecture {
-    local CPUName=$(uname -m)
-    local __arch=
-
-    if [[ "$(uname -s)" == "SunOS" ]]; then
-        CPUName=$(isainfo -n)
-    fi
-
-    case $CPUName in
-        i686)
-            __arch=x86
-            ;;
-        amd64|x86_64)
-            __arch=x64
-            ;;
-        armv7l)
-            __arch=arm
-            ;;
-        aarch64|arm64)
-            __arch=arm64
-            ;;
-        loongarch64)
-            __arch=loongarch64
-            ;;
-        riscv64)
-            __arch=riscv64
-            ;;
-        *)
-            echo "Unknown CPU $CPUName detected, configuring as if for x64"
-            __arch=x64
-            ;;
-    esac
-
-    echo "$__arch"
-}
-
 ################################################################################
 # Handle Arguments
 ################################################################################
 
-ARCH=$(check_cpu_architecture)
+source "$repoRootDir/eng/native/init-os-and-arch.sh"
+ARCH="$arch"
```

Comment:
nit: you can delete this and assign $arch to buildArch directly on line 55.

Reply:
OK, thanks